### PR TITLE
fix:[PLG-440] Fix shipper for CQ GCP and Wiz case

### PR DESCRIPTION
**Pull request by mkinchin-paladin**
## Description

Condition for CQ GCP case breaks data shipping for Wiz GCP data by setting `aacountname` to `null` because with Wiz we don't have all customer accounts and those data fields (such as `accountid` and `accountname`) are set in mapper.
### Solution
Added condition to the shipper so when account name is set - it's not get set again from the DB (crucial for Wiz).
Meanwhile updating all GCP mapping files in this [PLG PR 155](https://github.com/PaladinCloud/Plugins/pull/155) so they won't have '-' as a project name.

## Type of change

**Please delete options that are not relevant.**

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Did run shipper with changes - all project IDs where loaded properly for GCP (via regular CQ collection).
image

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] My commit message/PR follows the contribution guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java [hunk 1]
```diff
@@ -264,8 +265,9 @@ private void prepareDocs(Map<String, Map<String, String>> currentInfo, List<Map<
             } else if (entityInfo.containsKey("projectId")) {
                 entityInfo.put("accountid", entityInfo.get("projectId"));
             }
-            //For GCP CQ Collector accountName will be fetched from RDS using accountId
-            if ("gcp".equalsIgnoreCase(dataSource)) {
+            // For GCP CQ Collector accountName will be fetched from RDS using accountId
```
Comment (plyubich) [inline jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java:268]: this is CE code, can't expose CQ logic here. anyway to make this generic?

Comment (plyubich) [reply jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java:268]: probably just keep the comment for now.

File: jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/ExternalPolicies.java [hunk 2]
```diff
@@ -60,14 +60,19 @@ public static synchronized ExternalPolicies getInstance() {
      * @param dataSource the data source
      */
     public void uploadPolicyDefinition(String dataSource) {
-        LOGGER.info("Started upload policy definition for {}", dataSource);
+        LOGGER.info("Started upload external policies definition for {}", dataSource);
```
Comment (plyubich) [inline jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/ExternalPolicies.java:63]: same here, CE doesn't yet know about external policy concept, this naming should be generic.

**Review by plyubich — CHANGES_REQUESTED** [review]
please clean up comments to exclude exposing the technologies used by EE. Instead keep the naming generic

File: jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java (hunk 1 above)
Comment (mkinchin-paladin) [reply jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java]: I future yup, it should and will be generic. I guess, after we redefine how we work with accounts. All of those should have same naming for IDs and names, I see it like `accountID` and `accountName` for all types of sources, not like we have now - every single data source has its own naming for those.

File: jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/ExternalPolicies.java (hunk 2 above)
Comment (mkinchin-paladin) [reply jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/ExternalPolicies.java]: got it, fixed

File: jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java (hunk 1 above)
Comment (mkinchin-paladin) [reply jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java]: fixed

**Review by plyubich — DISMISSED** [review]

**Review by plyubich — APPROVED** [review]
LGTM

**Pull request merged**